setup.py

```diff
@@ -76,6 +76,7 @@ def get_extra_requires() -> dict:
             "async_timeout",
             "pettingzoo[classic]",
             "trueskill",
+            "envpool",
```

Comment:
delete this. Add a README.md in examples/envpool. And show how to install dependencies in the markdown.